single-pool/cli/src/main.rs

```diff
@@ -157,6 +164,61 @@ async fn command_initialize(config: &Config, command_config: InitializeCli) -> C
     ))
 }
 
+// reactivate stake account
+async fn command_reactivate(config: &Config, command_config: ReactivateCli) -> CommandResult {
+    let payer = config.fee_payer()?;
+    let pool_address = pool_address_from_args(
+        command_config.pool_address,
+        command_config.vote_account_address,
+    );
+
+    println_display(
+        config,
+        format!("Reactivating stake account for pool {}\n", pool_address),
+    );
+
+    let vote_account_address =
+        if let Some(pool_data) = config.program_client.get_account(pool_address).await? {
+            try_from_slice_unchecked::<SinglePool>(&pool_data.data)?.vote_account_address
+        } else {
+            return Err(format!("Pool {} has not been initialized", pool_address).into());
+        };
+
+    // the only reason this check is skippable is for testing, otherwise theres no reason
+    if !command_config.yolo {
+        let current_epoch = config.rpc_client.get_epoch_info().await?.epoch;
+        let pool_stake_address = find_pool_stake_address(&single_pool::id(), &pool_address);
+        let pool_stake_deactivated = quarantine::get_stake_info(config, &pool_stake_address)
+            .await?
+            .unwrap()
+            .1
+            .delegation
+            .deactivation_epoch
+            < current_epoch;
+
+        if !pool_stake_deactivated {
+            return Err("Pool stake account is not deactivated".into());
+        }
```

Comment:
Same as in the processor: I think we can relax the check to `<= current_epoch`, so that it's possible to reactivate something that was just deactivated in the same epoch